### DISCUSSION TITLE
Impossibile integrare Alexa in HA

### DISCUSSION THREAD

**Discussion by Manoel81** [indomus/forum#2845] [HUB: Home Assistant]
Buongiorno, ho seguito il più fedelmente la guida Indomus per l'integrazione di Alexa in HA. Tutto il percorso procede senza particolare intoppi fin quando non vado a lanciare il test della funziona Haaska, ossia l'ultimo step della guida. Qui mi appare un errore che recita:

```
Response
{
  "errorMessage": "Unable to import module 'haaska': No module named 'haaska'",
  "errorType": "Runtime.ImportModuleError",
  "requestId": "72f89900-cced-4f9d-a013-f103fb996c8f",
  "stackTrace": []
}
```
Quel "no module named haaska" a cosa fa riferimento, di preciso? Perché anche ricontrollando tutti i passaggi, più volte, non riesco a comprendere davvero cosa non trovi.

**Comment by KC-inDomus** [comment]
> Buongiorno, ho seguito il più fedelmente la guida Indomus per l'integrazione di Alexa in HA

purtroppo il punto è questo: hai sbagliato qualcosa al 99.9999999%. 
ribatti tutto punto per punto, e risolverai.

**Reply by Manoel81** [reply]
Sarà la decima volta che ripeto la guida, rileggendo più volte, segnandomi i passaggi che faccio, ecc... e continuo ad avere questo errore.

**Comment by Manoel81** [comment]
HO RISOLTO!!!

...seppur in maniera casuale e procedendo per tentativi, perché ero sicuro di aver fatto le cose per bene. Ma lo scrivo qui per dare un'imbeccata a coloro che si trovano nella mia stessa situazione (non pochi, a giudicare da Google).

In sostanza, escludendo un errore da parte mia perché davvero avevo fatto le cose con attenzione maniacale, mi sono concentrato su quello che poteva differire dalla guida. Mi riferivo alle pagine e a quelle voci leggermente diverse poiché aggiornatesi dopo l'uscita della guida stessa. Anche lì niente di fatto, quindi non rimaneva che una cosa: il file .zip da scaricare da Indomus.

Io ho il browser che decomprime gli archivi automaticamente, quindi il file zip diventava una normale cartella sul mio disco. Io la ricomprimevo sempre in formato zip senza nulla toccare per poi caricarlo come da guida. Ebbene, ho rimosso questa funzione dal mio browser, ho caricato il file zip così come scaricato dal sito Indomus e per magia il test è andato a buon fine!!

Adesso non mi resta che procedere oltre nella guida, sperando di riuscire ad arrivare fino in fondo ;)
